src/reducers/moon-phase.js

```diff
@@ -17,17 +15,17 @@ module.exports = (state, action) => {
     const payload = action.payload;
 
     switch (type) {
-        case MoonPhaseTypes.MOON_LOAD_BEGIN:
+        case 'MOON_LOAD':
             return Object.assign({}, state, {
                 isLoading: true
             });
-        case MoonPhaseTypes.MOON_LOADED:
+        case 'MOON_LOAD_FULFILLED':
             return Object.assign({}, state, {
                 isLoading: false,
                 moonId: payload.Index,
                 error: null
             });
-        case MoonPhaseTypes.MOON_LOAD_ERROR:
+        case 'MOON_LOAD_REJECTED':
```

Comment:
It's interesting to me that they don't offer helpers for the action naming convention they use!

Reply:
In their ["complex" example](https://github.com/pburtchaell/redux-promise-middleware/blob/master/examples/complex/reducers/application/support.js#L9), they would do:

```
case `${MoonPhaseTypes.MOON_LOAD}_REJECTED`:
```

You can also pass in what the [suffixes should be](https://github.com/pburtchaell/redux-promise-middleware/blob/master/src/index.js#L21).